pytest_mongodb/plugin.py

```diff
@@ -89,7 +89,7 @@ def load_fixtures(db, config):
         collection, ext = os.path.splitext(os.path.basename(file_name))
         file_format = ext.strip('.')
         supported = file_format in ('json', 'yaml')
-        selected = fixtures and collection in fixtures
+        selected = collection in fixtures if fixtures else True
```

Comment:
#10 has already been merged, which introduced the same logic. I would however rewrite this as
```python
selected = not fixtures or collection in fixtures
```
as proposed by @ndparker in the corresponding pull-request.